src/plist.c

```diff
@@ -915,6 +915,9 @@ Obj    FuncEmptyPlist( Obj self, Obj len )
             "you can replace <len> via 'return <len>;'" );
     }
 
+    if (INT_INTOBJ(len) < 0) {
+        ErrorMayQuit("<len> must be a non-negative small integer", 0L, 0L);
+    }
```

Comment:
It seems odd that one validation check allows me to enter a new value, the other doesn't. Why not just merge the two into a single one, anyway? I.e. `while ( ! intobj || value < 0)` ?